regulation/cms-0057-f/bulk-export-client-service/inMemoryStorage.bal

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/time;
+
+//This file represents the in-memory storage of the export tasks and polling events.
+
+final isolated map<ExportTask> exportTasks = {};
+
+isolated function addExportTasktoMemory(map<ExportTask> taskMap, ExportTask exportTask) returns boolean {
```

Comment:
We need to have interface with the defined set of functions to implement and InmemoryStorage is an implementation class of if. So that we can plug the implentation as needed.